app/views/smart_proxies/_disk_usage.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="card-pf">
+  <div class="card-pf-body">
+    <div class="progress-description">
+      <%= disk['header'] %>
```

Comment:
last comment -- adding a title attribute here would allow users to see all of the content places, sometimes I see this getting ellipsied when the content is too long:

![screenshot from 2016-01-26 10-52-27](https://cloud.githubusercontent.com/assets/293363/12585682/13f10606-c41b-11e5-9c4c-50304632f14d.png)